Projects/DbCLI/SearchDb.py

```diff
@@ -509,116 +511,122 @@ def poolFromCurs(curs, similarityMethod):
-parser.add_option('--smilesOut', '--smiOut', default='',
-                  help='export a smiles file with the matching molecules')
-parser.add_option('--nonchiralSmiles', dest='chiralSmiles', default=True, action='store_false',
-                  help='do not use chiral SMILES in the output')
-parser.add_option('--silent', default=False, action='store_true',
-                  help='Do not generate status messages.')
-
-parser.add_option('--zipMols', '--zip', default=False, action='store_true',
-                  help='read compressed mols from the database')
-
-parser.add_option('--pharm2DTableName', default='pharm2dfps',
-                  help='name of the Pharm2D fingerprints table')
-parser.add_option('--fdefFile', '--fdef',
-                  default=os.path.join(RDConfig.RDDataDir, 'Novartis1.fdef'),
-                  help='provide the name of the fdef file to use for 2d pharmacophores')
-parser.add_option('--gobbi2DTableName', default='gobbi2dfps',
-                  help='name of the Gobbi2D fingerprints table')
-
-parser.add_option(
-  '--similarityType', '--simType', '--sim', default='RDK', choices=supportedSimilarityMethods,
-  help='Choose the type of similarity to use, possible values: RDK, AtomPairs, TopologicalTorsions, Pharm2D, Gobbi2D, Avalon, Morgan. The default is %default')
-parser.add_option('--morganFpDbName', default='Fingerprints.sqlt',
-                  help='name of the morgan fingerprints database')
-parser.add_option('--morganFpTableName', default='morganfps',
-                  help='name of the morgan fingerprints table')
-parser.add_option('--morganFpColName', default='morganfp',
-                  help='name of the morgan fingerprint column')
-
-parser.add_option(
-  '--similarityMetric', '--simMetric', '--metric', default='',
-  choices=('tanimoto', 'dice', 'tversky', ''),
-  help='Choose the type of similarity to use, possible values: tanimoto, dice, tversky. The default is determined by the fingerprint type')
-parser.add_option('--tverskyA', default=0.5, type='float', help='Tversky A value')
-parser.add_option('--tverskyB', default=0.5, type='float', help='Tversky B value')
-parser.add_option(
-  '--simThresh', default=-1, type='float',
-  help='threshold to use for similarity searching. If provided, this supersedes the topN argument')
+
+def initParser():
+  """ Initialize the command line parser """ 
+  parser = argparse.ArgumentParser(usage='SearchDB [optional arguments] <sdffilename>',
+                                   description=_description,
+                                   formatter_class=argparse.RawDescriptionHelpFormatter)
+  
+  parser.add_argument('filename', nargs='?', help='File containg molecules for searching')
+  parser.add_argument('--version', action='version', version='%(prog)s ' + _version)
+
+  parser.add_argument(
+    '--dbDir', default='/db/camm/CURRENT/rdk_db',
+    help='name of the directory containing the database information. The default is %(default)s')
```

Comment:
I think we should replace the default here with the same as in the CreateDB script:
```
 parser.add_argument('--dbDir', default='', help='name of the directory containing the database information')
```

Reply:
Agreed. That old reference to `/db/camm` should be removed